Meta/serenity.sh

```diff
@@ -311,6 +311,14 @@ ensure_toolchain() {
 
 }
 
+confirm_rebuild_if_toolchain_exists() {
+    [ ! -d "$TOOLCHAIN_DIR" ] && return
+
+    read -rp "You already have a toolchain, are you sure you want to delete and rebuild one [y/N]? " input
+
+    if [[ "$input" != "y" && "$input" != "Y" ]]; then die "Aborted rebuild"; fi
```

Comment:
Nit-picky, but I'd rather not have the `if` and `then` clauses on one line; and no where else in this file is this style used.